File: ios/Light.m
//
//  Light.m
//
//  Created by 王亮 on 2019/2/28.
//  Copyright © 2019年 Facebook. All rights reserved.
//

#import "React/RCTViewManager.h"

@interface RCT_EXTERN_MODULE(Light, RCTViewManager)

+ (BOOL)requiresMainQueueSetup
{
  return YES;
}

RCT_EXPORT_VIEW_PROPERTY(isOn, BOOL)
RCT_EXPORT_VIEW_PROPERTY(onStatusChange, RCTDirectEventBlock)

@end

File: ios/Light.swift
//
//  Light.swift
//
//  Created by 王亮 on 2019/2/28.
//  Copyright © 2019年 Facebook. All rights reserved.
//

import Foundation

@objc(Light)
class Light: RCTViewManager {
    override func view() -> UIView! {
        return LightView()
    }
}
